releases/release-1.23/release-notes/maps/pr-101781-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 101781
+releasenote:
+  text: '[Errno -2] Name or service not known'
```

Comment:
This PR should not have a release note. Same issue as above. Could you remove this one as well?